backend/tests/cassettes/test_businesses_endpoint

```diff
@@ -0,0 +1,104 @@
+        K0tWayEGC97z6k8YYpki0XsaYoljRJjV/vP10+nnw+mlSWygC2AFPX9d9g0rt6I6FglP57rFIHaj
+        U3o/PoO89/JDqI1m88rHw/4jzGh3JGKfsMBBRizoPfgbW4u9kqRQPeccFueCzjDgdxIz8UGkvfS0
+        SyVfbrXUsf0RlVlJwbgucGYwB5nf9N34Zeo1UCihnUyg8SBaUowdOxwZPb7ZUmcLqKyk5Z7peFOh
+        qjrF13tmph2BRs1bnhkeZZVhw5Z1UnBxc1XP0iRuNoxSJpy7PlTGUFO/llkhFrw4PzSvHS8Ov6LM
+        367vHz4cTl3ebng238nl9NyiXdvQosETzs1KuMGSlF5QaSFEmymr3jhsF4oKxB+LBi56oLQMQZ8s
+        8m2eYiRo5C65kMDDJjF+W745oV7kRa3p2n6t0rCqgE7YrGjLLVuuoKxKOThLOcc2pzB5UiYUhjg7
+        yqEptT3Pbbm21sbN4TfpvllsOvNCCPc4bzaKDvpdLg5SpoFnKB9AwcZInEmbRpRR8v8GhpZgPuja
+        sxl2cX2QMqleyiGlrqodT5kFAgtszawwsCcQa/nuWQA32FlRQhte5xCu+6bNkwp3eeYVDiDkbOF4
+        sjwsjtlBO82av7qBFbTngs2xxYRxMXSb0UXsZldUaSM0qcWDNdsEhbM9Z+9QEne9qHtjzNW8nOdX
+        WCddibQU59kkSciGv0NPxE4AFLzvWtMeRdE69XzI92TlwQ99a0SHnUw7NpyL0h56TyVDhroO5ENV
+        LKrESwNG942onbi3bKQFcSxOOOgQrrokHHqGdGJq7coNjt6PuGnpE18M+FHkf83qotzFQo6VWiGE
+        xYqnu7zWTlyTKIeDdwWE117kTozS07eegMdlUrAyTzUrz29w9SG/x4CVpydegDABo1rQF2IAXOT4
+        j35VAZX2PZthZDS1mEkbH7EY6gtMeNXCG1FJBlMrzmYKfiUm1kYtVvVMvVQiXtQ5C4jU83+QzgAg
+        ZILqjlzJNcrhW7VRC2DEjn3dKIfNqY9p+QPxoizFgs24ilduwsbOLcThaz81rQiZaRZVBqI2bihQ
+        0ng3aSl1wQWqCIpiO1HI2I3Tikkht9uu0oG3E4VjO1w1L7erHCrxO7LEUZq85/tCHy1TRk7bx3vy
+        7huzIx3rg2Niu+blGtzSJdgFGtbLkxyIfUdflRUUlwauObayPrHljcSTodGhAy/yVJLZaRtcJbJ4
+        9WNKV5b2KC4NLETW3JUHtSw4zpyt8worXPl1uOcBMTANBsp7XYa2J5dEc8NvY+AxAaLstkssldXT
+        4PFYTqLYz3N+10Fs9+te/Cmw8+bE9wfX5yYFTFl2a/YloH2KWuKOLvFUwGtWJuLNMDE/FYONBAar
+        Va0hL1D7JyWZq7xO4ENXH0t+9YyhZEXTtuTR8+x1/HkF1gkUlmJ5VjRyrSXP7gkknBWoyKW817YN
+        75WpR9V64tRm90ty0cgeYQADrfU9x3kwiDxKpFP6M9CmFa4piP7FssErqoZajJSNJvXckWPAgWVr
+        xjFqnBGBY1xGzTjmInWZKW4OpuBZ4FuBond5YTh2N/bNVKUY73DEbIZtct/5+mQYyNq6GZT6rvo+
+        H2Tu9s5LO8zr2Qy2FBlUm47M9K37av6iAR+8MyfTliDw2iN3rZHfzN/YImH/iOMhsJuCp/hyTLJo
+        mYZtGa6x8FrpBMEyEDUsKZaW93yHov2s1eG3HzDJ4p+0wi/UnwpLJFZiSSivaGioFfFh76s//v7d
+        /wG2cT0R1j4AAA==
+    headers:
+      access-control-allow-origin: ['*']
+      age: ['0']
+      connection: [keep-alive]
+      content-encoding: [gzip]
+      content-type: [application/json; charset=UTF-8]
+      date: ['Tue, 31 Jan 2017 21:01:52 GMT']
+      etag: [W/"YWxwaGEuMjE0NjNfMl83MjBwdTdrZEpaVGZWbjByZHRhdHpHTFllbzNjLU3fv-_iktcm9ZJV8-PKmKj6yvNg4Q-gzip"]
+      last-modified: ['Tue, 31 Jan 2017 20:54:10 GMT']
+      server: [nginx]
+      x-socrata-region: [aws-us-east-1-fedramp-prod]
+      x-socrata-requestid: [7252216ty0w842twlbgdxulll]
+      x-soda2-data-out-of-date: ['false']
+      x-soda2-fields: ['["boro","building","camis","cuisine_description","dba","street","zipcode"]']
+      x-soda2-secondary-last-modified: ['Tue, 31 Jan 2017 20:54:10 GMT']
+      x-soda2-truth-last-modified: ['Tue, 31 Jan 2017 20:54:10 GMT']
+      x-soda2-types: ['["text","text","text","text","text","text","text"]']
+    status: {code: 200, message: OK}
+version: 1
```

Comment:
So this is actually what i get back as a cassette when i run the test of the businesses endpoint. 

Originally i thought it showed both requests and responses(the json returned by my endpoint, and the separate request to socrata with this string binary blob), but it really just added on the above to the previous cassette recording when the test used request.get(actual url) instead of client.get because i had been messing with the recording modes and `new_episodes` just added on this to the previous cassette.